src/main/java/frc/robot/commands/Shoot.java

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.subsystems.Shooter;
+
+public class Shoot extends CommandBase {
+  /** Creates a new Shoot. */
+  Shooter shooter;
```

Comment:
nit: fields should be `private final`
```suggestion
  private final Shooter shooter;
```